java/build.gradle

```diff
@@ -122,27 +122,31 @@ services.each { Service svc ->
     tasks.named(svc.id) { dependsOn deployModels, deployServices, deploySerializers, deployWebhookHandlers }
 }
 
-// Test small services generation
+// Test binlookup  generation
 tasks.named('binlookup') {
     doLast {
         assert file("${layout.projectDirectory}/repo/src/main/java/com/adyen/model/binlookup/Amount.java").exists()
         assert file("${layout.projectDirectory}/repo/src/main/java/com/adyen/service/BinLookupApi.java").exists()
+        // verify DefaultApi class no longer exists (it has been renamed to BinLookupApi)
         assert !file("${layout.projectDirectory}/repo/src/main/java/com/adyen/service/binlookup/DefaultApi.java").exists()
     }
 }
-// Test services generation
+// Test checkout generation
 tasks.named('checkout') {
     doLast {
         assert file("${layout.projectDirectory}/repo/src/main/java/com/adyen/model/checkout/Amount.java").exists()
         assert file("${layout.projectDirectory}/repo/src/main/java/com/adyen/service/checkout/PaymentsApi.java").exists()
     }
 }
-// Test webhooks generation
+// Test acswebhooks generation
 tasks.named('acswebhooks') {
     doLast {
         assert file("${layout.projectDirectory}/repo/src/main/java/com/adyen/model/acswebhooks/Amount.java").exists()
+        // verify no service package is created for a webhook
         assert !file("${layout.projectDirectory}/repo/src/main/java/com/adyen/service/acswebhooks").exists()
+        // verify no API class is created for a webhook
```

Comment:
Why should no API-class be created for webhook?

Suggestion: Verify that no API-class is created because webhook does not need a service-level class?

Reply:
Added a comment